json/pickler/src/main/scala/sttp/tapir/json/package.scala

```diff
@@ -0,0 +1,12 @@
+package sttp.tapir.json
```

Comment:
code organziation - maybe we should use a specific package, such as `sttp.tapir.json.pickler`?

Reply:
I was considering such an idea, it will be probably safer and clearer. I'll add a `pickler` package.